### ISSUE TITLE
Could you please provide the model pretrained on  Howto100M dataset?

### ISSUE THREAD

**Issue by sqiangcao99** [ArrowLuo/CLIP4Clip#13]
Hello, thanks for sharing your work. It's very inspiring. I don't have enough GPUs to pre-train the model on Howto100M. Could you please provide it？That will help me a lot to study your work.

**Comment by ArrowLuo** [comment]
Hi @sqiangcao99,
Thank you for your interest.  As our paper discussed, the pretrain is time-consuming. Our current pretrained weight has no significant advantage after finetuning (see Table 7 in the paper). So we have no plan to release the pretrained weight recently. Thanks.

Duplicate of #5
